scripts/core/services/pageTitle.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * PageTitle service
+ *
+ * PageTitle service is used to save and update the title of the page.
+ * The title will be 'Superdesk {- Url} {-Desk/Workspace}
+ * Url can be: Dashboard, Monitoring, Spike, Highlights, Search, Settings
+ * If Url is search then there won't be desk
+ *
+ */
+export default angular.module('superdesk.services.pageTitle', [])
+    .service('pageTitle', function() {
+        this.title = 'Superdesk';
+        this.url = '';
+        this.workspace = '';
+
+        this.setPageUrl = function(url) {
+            this.url = url;
+            setPageTitle(this.url, this.workspace);
+        };
+
+        this.setPageWorkspace = function(activeWorkspace) {
```

Comment:
Should we call this `setWorkspace` instead?

If it would be a global function, then `setPageWorkspace` would make sense, but when you take into account that it is called in the context of the name of this service (`pageTitle.setPageWorkspace`), then it's a bit duplicated.